src/rest/RequestHandler.js

```diff
@@ -77,6 +78,30 @@ class RequestHandler {
     });
   }
 
+  /*
+   * Determines whether the request should be queued or whether a RateLimitError should be thrown
+   */
+  async onRateLimit(request, limit, timeout, isGlobal) {
+    const { options } = this.manager.client;
+    if (options.disableRateLimitQueue) {
+      const rateLimitData = {
+        timeout,
+        limit,
+        method: request.method,
+        path: request.path,
+        route: request.route,
+        global: isGlobal,
+      };
+      const shouldThrow =
+        typeof options.disableRateLimitQueue === 'function'
+          ? await options.disableRateLimitQueue(rateLimitData)
+          : options.disableRateLimitQueue.some(route => rateLimitData.route.startsWith(`/${route}/`));
```

Comment:
So, the intended use case is specifying, say `channels` or `channels/:id` to the option? If so, hard coding it to be `/${route}/` is very limiting. Should simply be a `startsWith(route.toLowerCase())` (since routes are lowercased), to let us be very explicit about it (say you want to throw on `/gateway/bot` (also, the last `/` is invalid for all routes)